### DISCUSSION TITLE
20221208 Update - Network info page not displayed when Bluetooth enabled

### DISCUSSION THREAD

**Discussion by markblinkhorn** [rern/rAudio#1131] [Fixed Issues]
If I enable Bluetoooth I get the following displayed on the Network page:

Errors: Bad control character in string literal in JSON at position 331 

{

  "page"        : "networks"
, "activebt"    : 1
, "activeeth"   : true
, "activewl"    : true
, "camilladsp"  : false
, "connectedwl" : true
, "ipeth"       : ""
, "ipwl"        : "192.168.160.110"
, "listbt"      : [ {
  "mac"       : "14:4F:8A:4D:4B:CA"
, "name"      : "7P265Q2"
, "connected" : false
, "type"      : "SourceX
Sink"
} ]
, "listeth"     : false
, "listwl"      : [ {
	  "dbm"      : -60
	, "gateway"  : "192.168.160.254"
	, "ip"       : "192.168.160.110"
	, "ssid"     : "SI3_5GHz"
	},{
	  "ssid"     : "SI3"
	} ]
, "hostapd"     : false
, "hostname"    : "rAudio3"
}

I note similarity to https://github.com/rern/rAudio-1/discussions/1013 and tried pacman -U bluez-utils but that does not resolve the problem.

The 'X' in the text "type" : "SourceX" is highlighted in red (if that helps)

Any suggestions?

**Comment by rern** [comment]
Try:
`sed -i 's/\$info )/\$info | tr -d "\\n" )/' /srv/http/bash/settings/networks-data.sh`

**Reply by markblinkhorn** [reply]
Cool. That worked.
Thank you!
